core/templates/pages/blog-dashboard-page/blog-post-editor/blog-post-editor.component.html

```diff
@@ -0,0 +1,471 @@
+        <mat-card-subtitle class="sub-text general-text-format">
+          {{ 'I18N_BLOG_POST_EDITOR_LAST_SAVED_PREFIX' | translate }} {{ dateTimeLastSaved }}
+        </mat-card-subtitle>
+      </div>
+    </mat-card-header>
+    <mat-card-content class="text-center">
+      <mat-card-actions [ngClass]="{'small-screen-buttons-container': windowIsNarrow}">
+        <button mat-button class="mx-2 my-2"
+                [disabled]="(blogPostData.validate().length > 0) || !newChangesAreMade"
+                (click)="saveDraft()"
+                aria-label="Save blog post as draft."
+                [ngClass]="{'small-screen-button': windowIsNarrow}">
+          {{ 'I18N_BLOG_POST_EDITOR_SAVE_DRAFT_BUTTON' | translate | uppercase }}
+        </button>
+        <button mat-button class="mx-2 my-2 publish-button"
+                [disabled]="isPublishButtonDisabled()"
+                (click)="publishBlogPost()"
+                aria-label="Publish blog post."
+                [ngClass]="{'small-screen-button': windowIsNarrow}">
+          {{ 'I18N_BLOG_POST_EDITOR_PUBLISH_BUTTON' | translate | uppercase }}
+        </button>
+        <div *ngIf="!windowIsNarrow" class="mt-4 text-left">
+          <span class="tag-heading general-text-format">
+            {{ 'I18N_BLOG_POST_EDITOR_TAGS_HEADING' | translate}}
+          </span>
+          <span class="sub-text tag-limit-text general-text-format">
+            {{ 'I18N_BLOG_POST_EDITOR_TAGS_LIMIT_PREFIX' | translate}} {{maxAllowedTags}}
+          </span>
+          <span class="sub-text d-block tag-limit-text general-text-format">
+            {{maxAllowedTags-blogPostData.tags.length}} {{ 'I18N_BLOG_POST_EDITOR_TAGS_LIMIT_SPECIFICATION_SUFFIX' | translate}}
+          </span>
+        </div>
+        <div *ngIf="!windowIsNarrow">
+          <mat-button-toggle-group aria-label="Tags to classify blog post."
+                                   [value]="blogPostData.tags"
+                                   *ngFor="let tag of defaultTagsList"
+                                   multiple="true"
+                                   class="mx-2 my-2 tag">
+            <mat-button-toggle class="text-wrap"
+                               (change)="onTagChange($event.value)"
+                               [value]="tag"
+                               [disabled]="!((blogPostData.tags).includes(tag)) && (blogPostData.tags.length === maxAllowedTags)">
+              {{ tag }}
+            </mat-button-toggle>
+          </mat-button-toggle-group>
+        </div>
+        <button mat-button
+                class="d-block delete-button mx-auto mt-2"
+                (click)="deleteBlogPost()"
+                aria-label="Delete blog post."
+                [ngClass]="{'small-screen-button': windowIsNarrow}">
+          {{ 'I18N_BLOG_POST_EDITOR_DELETE_BUTTON' | translate | uppercase }}
+        </button>
+      </mat-card-actions>
+    </mat-card-content>
+  </mat-card>
+</div>
+<style>
+  .author-username {
```

Comment:
Can all these css classes be wrapped under `.blog-post-editor`?

Reply:
It is not required as in angular the css does not bleed to other components. Thus it will not have any effect on CSS of other classes.